### ISSUE TITLE
Problemas Servidor

### ISSUE THREAD

**Issue by arturokunder** [IIC3103/2019-proyecto#85]
Al parecer hubieron algunos problemas en el servidor de desarrollo.
Resetié todo para que puedan seguir probando.
También he estado revisando los temas levantados de la entrega 2 (#74)

Revisaré bien el servidor y más tarde les comento más en detalle de ambos temas.

**Comment by arturokunder** [comment]
En relación a la entrega 2, estuve revisando las estadísticas del sistema y he llegado a las siguientes conclusiones:
1. La BD se cayó el sábado tipo 7:30 am (UTC)
1. El día anterior, hubo problemas en la fabricación (#74).

Por los dos temas anteriores, las accioes a tomar serán:
1. Re-revisaré la entrega pasada tomando en cuenta el punto 2. El punto ya estaba considerado en la corrección.
1. Realizaré las correcciones correspondientes al sistema para que no se repitan estos problemas.
1. Les publicaré las estadísticas de producción que obtuve de la BD para que contrasten con sus datos.
Los que me han enviado mails respecto a la entrega 2, no los he contestado aun a la espera de tener toda la información anterior.

Respecto a los problemas de hoy con la API (#82, #84):
1. Ahora está todo funcionando correctamente (por favor avisarme a la brevedad si no es así)
1. El monitoreo (y reset de producción) se pospondrá 24 hrs para que puedan recuperar el tiempo y poder hacer los cambios  con el fin que el sistema esta vez no falle.

Que tengan un excelente fin de semana,

Arturo

**Comment by cafuentes1** [comment]
No me está funcionando fabricar sin pago en prod, solo en dev. Grupo 4.

**Comment by arturokunder** [comment]
@cafuentes1 producción aun no se habilita

**Comment by tomvillage** [comment]
Profesor nuevamente esta fallando fabricar en dev

**Comment by svergara1** [comment]
@arturokunder

**Comment by arturokunder** [comment]
Revisando.

**Comment by arturokunder** [comment]
Listo, solucionado. 
Ya se está produciendo. Si no les llegaron los productos, les pido que reenvíen a producir

**Comment by vbarros2** [comment]
Se volvió a caer... @arturokunder

**Comment by arturokunder** [comment]
Si, ya está. Justo estaba trabajando en la bd

**Comment by atbusch** [comment]
@arturokunder No nos está funcionando ninguna request en produccion y en desarrollo si nos funcionaban. Tenemos Response 500 en todas las de produccion. Somos el grupo 2

**Comment by arturokunder** [comment]
Si, está correcto.
Producción se habilitará a las 14:00 (ver #86)

**Comment by leylayunis** [comment]
@arturokunder Estamos teniendo demoras excesivas en la fabricación de productos que va a ser que nos sea imposible cumplir con las ordenes de compra. Ayer en la tarde nos cambiamos a producción y la fabricación funcionaba perfecto pero hoy durante la mañana comenzamos con las demoras.
Acabo de mandar a fabricar 10 de 1001 y me dice que va a llegar en casi 5 horas .... 
<img width="441" alt="Captura de Pantalla 2019-06-25 a la(s) 13 09 28" src="https://user-images.githubusercontent.com/17606660/60118511-85d1bd00-974a-11e9-8e4b-f87beec5a9c1.png">

**Comment by arturokunder** [comment]
@leylayunis 
Es completamente normal. Tal como sale en el excel de productos, el tiempo esperado de producción es de 240 mins, lo que significa que sobre este número hay cierta variabilidad.
Los atrasos en la fábrica los definimos como el atraso sobre la fecha `disponible` publicada al llamar al servicio por sobre la fecha real en que llegan los productos

**Comment by JoseRomeroi** [comment]
@arturokunder Profe, pal monitoreo va a resetear las estadisticas b2b ?

**Comment by arturokunder** [comment]
no. no está considerado. el último reset fue ayer (como se detalla en #86)

**Comment by JoseRomeroi** [comment]
@arturokunder Pero si el monitoreo parte a las 18:00 de hoy, quiere decir que para nuestra nota solo cuentan las B2B que nos lleguen a partir de las 18:00 ??

**Comment by arturokunder** [comment]
Tienes razón. Así dice el enunciado.
Pensé que decía que el monitoreo comenzaba con el último reset.
No consideraré las oc recibidas antes de las 18:00:00 de hoy.

**Comment by leylayunis** [comment]
@arturokunder y van a descontar los costos de pulmón hasta antes de hoy a las 18:00?
